src/LanguageManager/LanguageManagerServiceProvider.php

```diff
@@ -87,18 +94,19 @@ public function register( Container $container ) {
 	public function bootstrap( Container $container ) {
 
 		$container['multilingualpress.language_manager_settings_page']->register();
+		$labels = $container['multilingualpress.language_meta_labels'];
 
 
 		add_action( LanguageManagerSettingsPageView::ACTION_CONTENT_DISPLAY, function() use (
-			$container
+			$container, $labels
```

Comment:
*When* we break items, we break each onto its own line. However, we have a line length of 120, so you can put all of that into a single line.